source/Sagan-RDBMS-Tests/SingleSessionRDMBSRepositoryTest.class.st

```diff
@@ -215,3 +224,45 @@ SingleSessionRDMBSRepositoryTest >> testMoreThanOneRepository [
 		do: [ :spaceship | self assert: spaceship name equals: 'UFO 54-40' ]
 		else: [ self fail ]
 ]
+
+{ #category : #'tests - querying' }
+SingleSessionRDMBSRepositoryTest >> testSpaceshipQueryingForPositiveFirepower [
+
+	self setUpRepositoryWithNoConflictChecking.
+
+	ships store: ( Spaceship withoutFirepowerNamed: 'Solvalou' ).
+	ships store: ( Spaceship named: 'Solgrado' firepower: 7650 ).
+	ships store: ( Spaceship withUnlimitedFirepowerNamed: 'Zeodalley' ).
+
+	self
```

Comment:
I don't know why the tests are repeated but the same comment as before applies here.